### DISCUSSION TITLE
Expanding the left side bar?

### DISCUSSION THREAD

**Discussion by sam-lindquist** [academicpages/academicpages.github.io#1996] [Q&A]
How could I increase the size of the left side bar? I'm thinking it would be nice to increase the width and increase the font sizes of all the stuff in it (e.g. name, email, and any other info).

Thanks very much for any advice!

**Comment by rjzupkoii** [comment]
This might be the same question that was brought up in #2014?

**Reply by sam-lindquist** [reply]
Hello,

It looks like a similar question, yes. Thank you!

Sam
